### ISSUE TITLE
JibImageBuildServiceBuildTest is failing on windows

### ISSUE THREAD

**Issue by Sintivrousai** [eclipse-jkube/jkube#3156]
Related to https://github.com/eclipse-jkube/jkube/issues/1338

JibImageBuildServiceBuildTest is failing on windows.

[ERROR] Errors:
[ERROR]   JibImageBuildServiceBuildTest.build_withLayersAndArtifact_shouldPerformJibBuild:194 ? JKubeService Error when building JIB image

**Comment by rohanKanojia** [comment]
@Sintivrousai : Are you working on this issue? Shall I assign it to you?

**Comment by Sintivrousai** [comment]
Yes, assign me please

**Comment by Sintivrousai** [comment]
I think the test is okey and the problem was due to the cache of my device. So i should close the issue, right?

**Comment by rohanKanojia** [comment]
@Sintivrousai : Does it work if you clear the cache? Which cache are you referring to in your comment? Is it  maven cache or docker cache?

**Comment by Sintivrousai** [comment]
Yes, it worked after i cleared it. I am not sure about the type of cache. I used the link of the surefire report (..\AppData\Local\Temp\jib-core-application-layers-cache\layers\12c9729fa63b46c247fedf5ff795775b9edd1da6c55999ff040613577ad2d3cd\c0fdebab48fd4b0af65977804fd7b164856da464f20756381594d1bac9e63b85)

**Comment by rohanKanojia** [comment]
@Sintivrousai : Are you still facing the issue? Shall we close this issue?
